sdk/interfaces/__tests__/logout.test.js

```diff
@@ -0,0 +1,30 @@
+import { logout } from '../index';
+import makeRequest from '../../requests';
+import REQUEST_TYPES from '../../utils/constants';
+
+jest.mock('../../requests');
+
+afterEach(() => jest.clearAllMocks());
+
+describe('logout', () => {
+  it('calls logout and works correctly', async () => {
+    const postLogoutRedirectUri = 'postLogoutRedirectUri';
+    makeRequest.mockReturnValue(Promise.resolve(postLogoutRedirectUri));
+    const response = await logout();
+    expect(makeRequest).toHaveBeenCalledTimes(1);
+    expect(makeRequest).toHaveBeenCalledWith(REQUEST_TYPES.LOGOUT);
+    expect(response).toBe(postLogoutRedirectUri);
+  });
+  it('calls logout and fails', async () => {
```

Comment:
```suggestion
  });
  
  it('calls logout and fails', async () => {
```
Para que queden separado cada it.